src/DEFCON-5.sol

```diff
@@ -111,23 +116,17 @@ contract DssSpell {
         bytes32[] memory ilks = registry.list();
 
         for (uint i = 0; i < ilks.length; i++) {
-            // skip the rest of the loop for the following ilks:
-            //
-            if (
-                ilks[i] == "USDC-A"   ||
-                ilks[i] == "USDC-B"   ||
-                ilks[i] == "TUSD-A"   ||
-                ilks[i] == "PAXUSD-A" ||
-                ilks[i] == "GUSD-A"   ||
-                ilks[i] == "PSM-USDC-A"
+            if (ilks[i] == "PSM-USDC-A" ||
+                ilks[i] == "UNIV2DAIUSDC-A"
```

Comment:
This is good to get the tests passing for now.  Once we fix this in this weeks exec, we can remove this and the tests should pass.